### PR TITLE
test: cy.getByTestId

### PR DESCRIPTION
**Pull request by zzmp**
## Description

Adds a cypress command to simplify getting an element by data-testid: `cy.getByTestId`.

_Relevant docs:_ https://www.notion.so/uniswaplabs/E2E-Infrastructure-Tracker-3739b243246f41aa98de4df5e4c880b7

### Automated testing

- [x] Integration/E2E test

### CODE REVIEW

File: cypress/e2e/token-explore.test.ts [hunk 1]
```diff
@@ -1,38 +1,34 @@
...
-
 describe('Token explore', () => {
   before(() => {
     cy.visit('/')
   })
 
   it('should load token leaderboard', () => {
     cy.visit('/tokens/ethereum')
-    cy.get(getTestSelectorStartsWith('token-table')).its('length').should('be.greaterThan', 0)
+    cy.get('[data-testid^=token-table]').its('length').should('be.greaterThan', 0)
```
Comment (JFrankfurt) [inline cypress/e2e/token-explore.test.ts:8]: nice

File: cypress/support/e2e.ts [hunk 2]
```diff
@@ -37,9 +37,15 @@ declare global {
        */
       userState?: Partial<UserState>
     }
+    interface Chainable {
+      /** Get one or more DOM elements by data-testid attribute. */
+      getByTestId<E extends Node = HTMLElement>(selectorId: string): Chainable<JQuery<E>>
+    }
```
Comment (JFrankfurt) [inline cypress/support/e2e.ts:43]: it feels like this should be done by the command

**Review by JFrankfurt — APPROVED** [review]
lgtm

File: cypress/support/e2e.ts (hunk 2 above)
Comment (zzmp) [reply cypress/support/e2e.ts:43]: Cypress requires that you type and add a command separately, as is done here.

**Pull request closed without merging**